src/base/empty_lambda.h

```diff
@@ -11,7 +11,7 @@ class EmptyLambda {
 
   template <class... Args>
   ReturnType operator()(Args&&...) {
-    return ReturnType();
+    return value_;
```

Comment:
Now it returns a value provided by user, not a default value for type.
I use it with a |true| value, but without this fix it returns |false|.